containers/home.tsx

```diff
@@ -1,66 +1,92 @@
-import React from 'react';
-import Head from 'next/head'
-import styles from '../styles/Home.module.css'
+import React, { useState, useEffect } from 'react';
+import Head from 'next/head';
+
+// Components
+import ListCardVideo from '../components/listCardVideo/listCardVideo';
+import ArrowPrevSVG from '../components/svg/arrowPrevSVG';
+import ArrowNextSVG from '../components/svg/arrowNextSVG';
+
+// Style
+import styles from '../styles/Home.module.css';
+interface iVideo {
+  cards?: [{ title: string; alt: any; src: any }];
+}
+
+const Home: React.FC<iVideo> = () => {
+  const [viewNextVideo, setViewNextVideo] = useState(0);
+  const [showVideos, setShowVideos] = useState([]);
+
+  const cards = [
+    { title: 'Course 1', alt: 'img1', src: '1' },
+    { title: 'Course 2', alt: 'img2', src: '2' },
+    { title: 'Course 3', alt: 'img3', src: '3' },
+    { title: 'Course 4', alt: 'img4', src: '4' },
+    { title: 'Course 5', alt: 'img5', src: '5' },
+    { title: 'Course 6', alt: 'img6', src: '6' },
+  ];
+
+  useEffect(() => {
+    const videosToShow = [];
+
+    for (let i = viewNextVideo; i <= viewNextVideo + 3; i++) {
```

Comment:
Why `viewNextVideo + 3`? Do you think we could find a way to make this more clear?

Reply:
I think that we can rename the state for example as indexVideo / setIndexVideo, what do you thing?

Reply:
In this case, my question was more like why does the exit condition need to be exactly `viewNextVideo + 3`? Is there any particular reason behind that?

Reply:
Ok thanks, in this case, because we must show 4 videos like is in the wireframes, and for each interaction, we must have four in total

Reply:
Oh, got it! In this case, that `3` can be replaced with a constant like `MAX_VIDEOS_TO_SHOW` or something similar.

It's a good practice to always avoid [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)).

Reply:
Thank you so much for the suggestion